sdk/samples/iot/CMakeLists.txt

```
@@ -85,6 +85,17 @@ target_link_libraries(paho_iot_pnp_sample
     az::iot::pnp
 )
 
+# PnP Sample
+add_executable (paho_iot_pnp_with_provisioning_sample
+  ${CMAKE_CURRENT_LIST_DIR}/paho_iot_pnp_with_provisioning_sample.c
```

SAS sample is named this paho_iot_provisioning_sas_sample.c
```suggestion
  ${CMAKE_CURRENT_LIST_DIR}/paho_iot_provisioning_pnp_sample.c
```

---

I think I'm going to push back on this one. I think the naming of the samples is based on what feature is being highlighted. In the `paho_iot_provisioning_sas_sample.c`, the star of the show is provisioning and the provisioning client. In the `paho_iot_hub_` samples, the star is the hub client. And with `paho_iot_pnp_`, it is showcasing plug and play functionality. I think the root point of this sample is plug and play with possibly different auth scenarios. So there could be in theory some `paho_iot_pnp_sas_sample.c`. Just like provisioning has `paho_iot_provisioning_sample` and `paho_iot_provisioning_sas_sample`